extra/utils.py

```diff
@@ -136,7 +140,7 @@ def fake_torch_load_zipped(fb0, load_weights=True, multithreaded=True):
       def load_weight(k, vv):
         with myzip.open(f'{base_name}/data/{k}') as myfile:
           for v in vv:
-            load_single_weight(v[2], myfile, v[3], v[4], v[0], v[5], mmap_allowed=True)
+            load_single_weight(v[2], myfile, v[3], v[4], v[0], v[5], mmap_allowed=True, load_dtype=load_dtype)
```

Comment:
I think would be cleaner to just have 2 calls to `post_process` (after each `load_single_weight`). Here and in 
https://github.com/geohot/tinygrad/blob/0fc4cf72a2261fff7ef432ed9eb9bd003459dcd9/extra/utils.py#L184